api/src/user/controllers/auth.controller.ts

```diff
@@ -66,7 +73,13 @@ export class BaseAuthController {
   logout(
     @Session() session: ExpressSession,
     @Res({ passthrough: true }) res: Response,
+    @Headers() headers: Record<string, string>,
   ) {
+    const parsedCookie = cookie.parse(headers['cookie']);
+    const sessionCookie = encodeURIComponent(
+      String(parsedCookie[config.session.name] || ''),
+    );
+    this.eventEmitter.emit('hook:websocket:session_data', sessionCookie);
```

Comment:
```suggestion
    this.eventEmitter.emit('hook:user:logout', sessionCookie);
```
I suggest we do the api part in another PR.